### PR TITLE
add cnn sentiment model for dygraph

### PR DESCRIPTION
**Pull request by wopeizl**

### CODE REVIEW

File: dygraph/sentiment/main.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+from paddle.fluid.dygraph.base import to_variable
+import nets
+import reader
+from utils import ArgumentGroup
+
+DATA_PATH = "./senta_data/"
+CKPT_PATH = "./save_models/"
+VOCAB_PATH = DATA_PATH + "word_dict.txt"
+
+parser = argparse.ArgumentParser(__doc__)
```
Comment (JiabinYang) [inline dygraph/sentiment/main.py:26]: remove useless arguments

File: dygraph/sentiment/nets.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        return x
+
+
+class CNN(fluid.dygraph.Layer):
+    def __init__(self, name_scope, dict_dim, batch_size, seq_len):
+        super(CNN, self).__init__(name_scope)
+        self.dict_dim = dict_dim
+        self.emb_dim = 128
+        self.hid_dim = 128
+        self.hid_dim2 = 96
```
Comment (JiabinYang) [inline dygraph/sentiment/nets.py]: better to call this `fc_hid_dim`?

File: dygraph/sentiment/nets.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        self.batch_size = batch_size
+        self._conv2d = Conv2D(
+            self.full_name(),
+            num_channels=num_channels,
+            num_filters=num_filters,
+            filter_size=filter_size,
+            padding=[0, 1],
+            use_cudnn=use_cudnn)
+
+        self._pool2d = Pool2D(
```
Comment (JiabinYang) [inline dygraph/sentiment/nets.py]: remove this

File: dygraph/sentiment/nets.py (hunk 2 above)
Comment (wopeizl) [reply dygraph/sentiment/nets.py]: ok

File: dygraph/sentiment/main.py (hunk 1 above)
Comment (wopeizl) [reply dygraph/sentiment/main.py:26]: ok

File: dygraph/sentiment/nets.py (hunk 3 above)
Comment (wopeizl) [reply dygraph/sentiment/nets.py]: ok

**Review by junjun315 — APPROVED** [review]
LGTM

**Pull request merged**